### PR TITLE
add Express.js example for data stream pipe

### PR DESCRIPTION
**Pull request by arturojain**
#10 #12  #13

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,16 +29,23 @@ function listObjectInner() {
...
         this.awsConfig = awsConfig;
-        AWS.config.update({
-            accessKeyId: awsConfig.accessKeyId,
-            secretAccessKey: awsConfig.secretAccessKey,
-            region: awsConfig.region
-        });
-        this.s3bucket = new AWS.S3({
-            params: {
-                Bucket: this.awsConfig.bucket
+        AWS.config.getCredentials(function (err) {
```
Comment (DanielHindi) [inline index.js]: Can you explain the reasoning behind this change

Comment (arturojain) [inline index.js]: @DanielHindi when using aws-s3-zipper on AWS EC2 the credentials are already configured, or if you have configured AWS previously or using environment variables there is no need to configure it again.

Comment (arturojain) [inline index.js]: @DanielHindi sorry for the pull requests errors, I am very new to using them.

**Comment by arturojain** [comment]
I need to fix this pull request, I will close it to improve my contribution.

**Pull request closed without merging**